modules/custom/az_card/templates/az-card.html.twig

```diff
@@ -18,17 +19,26 @@
 
 <div{{ attributes }}>
   {% if media %}
-    {{ media }}
+    <div class="card border-0 overflow-hidden">
+      {{ media }}
+      {% if title and title_style == 'title-on-image' %}
+        <div class="card-img-overlay p-0 d-flex flex-column">
+          <h3 class="h5 bold mt-auto card-title bg-gradient-black text-white mb-0 p-card pt-8">{{ title }}</h3>
+        </div>
+      {% endif %}
+    </div>
+  {% endif %}
+  {% if title_style == 'default' or (title_style == 'title-on-image' and (body or link)) %}
+    <div class="card-body d-flex flex-column">
```

Comment:
Follow up ticket would be to add the `card-body` class here only if these conditions apply:
- Title on image style is selected AND content is in the body OR link title fields.
- Title within card style is selected AND content is in the title OR body OR link title fields.